api-2.2/pom.xml

```diff
@@ -11,9 +11,20 @@
 	<name>Core Apps Module API 2.2</name>
 	<description>API project for CoreApps</description>
 	<properties>
```

Comment:
There seem to be a lot of changes to dependencies here... can you provide a summary of what you are changing and why?

Reply:
Oh, yeah. Doing this. I brief, these dependencies are necessary for the context sensitive tests.

Reply:
@Ruhanga the they should be test-scoped:
```xml
<scope>test</scope>
```

Reply:
Wouldn't the dependencies be needed for later usage by this sub-project @mks-d ?

Reply:
@Ruhanga until they are not needed outside of the tests they should remain test-scoped.